Document the beta secrets manager (#3445)
chef/chef-web-docs: content/infra_language/secrets.md

+++
title = "Chef Infra Language: Secrets"
draft = false

gh_repo = "chef-web-docs"

[menu]
  [menu.infra]
    title = "Secrets Management Integrations"
    identifier = "chef_infra/infra_language/secrets.md Secrets Management Integrations"
    parent = "chef_infra/infra_language"
+++

Chef Infra Client 17.5 and later includes a beta Secrets Management Integration helper. This helper allows accessing secrets from the following secrets management systems within your Infra recipes or resources:

- AWS Secrets Manager
- Akeyless Vault
- Azure Key Vault
- HashiCorp Vault

{{< note >}}

> This helper is a beta release, meaning it under active development and may change over time. If you have feedback on any functionality you'd like to see in this helper please reach out at secrets_management_beta@progress.com.

Reviewer: ```suggestion
This helper is a beta release. It is under active development and may change. We welcome your feedback at secrets_management_beta@progress.com.
```
